staff/PacoGD/blog/links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
**staff/PacoGD/blog/links/index.html!!!**

Watch out!! better use another name for a non principal html file.

Reply:
Done!